### PR TITLE
feat: beinleumi scrapper - balance support

### PR DESCRIPTION
**Pull request by LiranBri**

### CODE REVIEW

File: src/scrapers/base-beinleumi-group.ts
Comment (eshaham) [inline src/scrapers/base-beinleumi-group.ts:279]: can you move this above the function which calls it?

File: src/scrapers/base-beinleumi-group.ts
Comment (eshaham) [inline src/scrapers/base-beinleumi-group.ts:284]:
```suggestion
  return getAmountData(balanceStr);
```

**Review by eshaham — CHANGES_REQUESTED** [review]
Hi @LiranBri,
Thanks for contributing.
Added small comments.
Also, notice that your PR doesn't pass the build - can you take a look?

File: src/scrapers/base-beinleumi-group.ts
Comment (LiranBri) [reply src/scrapers/base-beinleumi-group.ts:279]: I can but In my opinion it's good practice for clean code to put the higher-level function below the lower level, because then you can read the code from top to bottom.
reading first the lower level implementation makes no sense on its own because there is no context. so you always want to read the higher-level code first, and only then dig down to the lower level implementations (the sub-function it triggers)

**Comment by LiranBri** [comment]
> Also, notice that your PR doesn't pass the build - can you take a look?
where can I see that? can you share a link or the error?
I see just list of build pipelines with status they are waiting for status change before they could run (I guess waiting for PR approval of maintainer)

File: src/scrapers/base-beinleumi-group.ts
Comment (eshaham) [reply src/scrapers/base-beinleumi-group.ts:279]: @LiranBri that's a matter of taste 😉  plus that's how we have the lint rules set up for the entire project at the moment

**Comment by eshaham** [comment]
> where can I see that? can you share a link or the error?
you should see it now - you can also run `npm run lint` locally to see the lint errors

File: src/scrapers/base-beinleumi-group.ts
Comment (LiranBri) [reply src/scrapers/base-beinleumi-group.ts]: IMO its a bad practice, but your project, your call :)
fixed

**Pull request merged**
